image/tarexport/load.go

```diff
@@ -421,9 +422,9 @@ func checkCompatibleOS(imageOS string) error {
 	if runtime.GOOS != "windows" && imageOS == "windows" {
 		return fmt.Errorf("cannot load %s image on %s", imageOS, runtime.GOOS)
 	}
-	// Finally, check the image OS is supported for the platform.
-	if err := system.ValidatePlatform(system.ParsePlatform(imageOS)); err != nil {
-		return fmt.Errorf("cannot load %s image on %s: %s", imageOS, runtime.GOOS, err)
+	
+	if _, err := platforms.Parse(imageOS); err != nil {
```

Comment:
Hm, looks like there's a `gofmt` issue; 

```
11:27:52 image/tarexport/load.go:1::warning: file is not gofmted with -s (gofmt)
11:27:52 image/tarexport/load.go:1::warning: file is not goimported (goimports)
```

Reply:
Ah, yes, I am thinking how to add a automatic hook to trigger the `gofmt` for all the changed sourced code... Will update it soon, but probably need your time to rebuild some nodes 😄